src/frontend/components/TranscriptReader/index.tsx

```diff
@@ -36,12 +36,17 @@ export const TranscriptReader = ({ transcript }: TranscriptReaderProps) => {
   useEffect(() => {
     if (!transcriptWrapperRef.current) return;
 
-    transcriptWrapperRef.current
-      .querySelector(`.sentence-active`)
-      ?.scrollIntoView({
-        behavior: 'smooth',
-        block: 'center',
-      });
+    const target = transcriptWrapperRef.current.querySelector(
+      `.sentence-active`,
+    ) as HTMLSpanElement;
+
+    if (target) {
+      // Follow active sentence to be always visible.
+      transcriptWrapperRef.current.scrollTop =
```

Comment:
That's work. I just use `scrollTo` method to keep the smooth scrolling effect.

```suggestion
      transcriptWrapperRef.current.scrollTo({
        top: target.offsetTop -
        (transcriptWrapperRef.current.offsetTop +
          transcriptWrapperRef.current.offsetHeight / 2),
        behavior: 'smooth',
      })
```

Reply:
Done